testing/file_test/autoupdate.cpp

```diff
@@ -341,6 +347,10 @@ auto FileTestAutoupdater::Run(bool dry_run) -> bool {
     ++non_check_line_;
   }
 
+  // This will result in all check lines (and only check lines) being
+  // added to the split.
```

Comment:
Only if `autoupdate_split_` was initially true, right?

Would it work to put this inside the loop above, right before the `break`? It seems like it might be clearer in that context.

Reply:
> Only if `autoupdate_split_` was initially true, right?

Modifying comment.

> Would it work to put this inside the loop above, right before the `break`? It seems like it might be clearer in that context.

It's mainly to control the behavior of FinishFile, so I lean towards it being adjacent to the affected function call...